### PR TITLE
Feature/yearly

### PR DESCRIPTION
**Pull request by Xayrulloh**

### CODE REVIEW

File: config/database.ts [hunk 1]
```diff
@@ -82,6 +82,11 @@ const PrayTime = new mongoose.Schema(
       required: true,
       type: Number,
     },
+    month: {
+      required: true,
+      type: Number,
+      default: 2,
```
Comment (eugene0928) [inline config/database.ts:88]: no need for required if there is default value set

**Review by eugene0928 — APPROVED** [review]

File: cron/cron.ts [hunk 2]
```diff
@@ -14,57 +14,59 @@ import cron from 'node-cron'
...
-              regionId: regionIds[i],
-              day: dayNumber,
-              fajr: times[0],
-              sunrise: times[1],
-              dhuhr: times[2],
-              asr: times[3],
-              maghrib: times[4],
-              isha: times[5],
-            })
+  for (let month = 1; month <= 12; month++) {
```
Comment (toshmukhamedov) [inline cron/cron.ts:26]: hamma joyda month + 1 qilib ishlatmasdan shunchaki monthni 0 dan boshlasak qanday bo’ladi?

**Review by toshmukhamedov — CHANGES_REQUESTED** [review]
Iloji bo’lsa monthlyni saqlab qoling (ishlatilmasa ham)

File: cron/cron.ts (hunk 2 above)
Comment (Xayrulloh) [reply cron/cron.ts:26]: image

as u can see we can't cause its starting from 1

**Review by toshmukhamedov — APPROVED** [review]

**Pull request merged**